docs/wifi-setup.md

```diff
@@ -18,8 +18,112 @@ There are several ways to connect your Anthias instance to Wi-Fi. You can either
 configure the Wi-Fi settings via the captive portal or you can run the `raspi-config`
 on your terminal.
 
+The `raspi-config` and `nmcli` commands will only work on devices running a
+Raspberry Pi OS Lite as those running balenaOS gives users less control over
+some configs.
+
+### Using `nmcli`
```

Comment:
Might be better to recommend using `nmtui` instead as it is a lot more user friendly.